src/tokens/color/status.json

```diff
@@ -0,0 +1,14 @@
+{
+  "color": {
+    "status": {
+      "error_dark": {
+        "value": "#bc1919",
+        "comment": "For errors, intended to be used for the foreground when used on the light theme"
```

Comment:
Now that I'm reading them, I'm not sure these comments are actually providing helpful info. Maybe we can remove them entirely for now.